### PR TITLE
Update page in APIM release note section “20200530_apip_relnotes”

### PR DESCRIPTION
**Pull request by jowens-axway**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/en/docs/apim_relnotes/20200530_apip_relnotes.md [hunk 1]
```diff
@@ -18,9 +18,29 @@ API Portal is available as a software installation or a virtualized deployment i
 
 ## New features and enhancements
 
-<!-- Add the new features here -->
+The following new features and enhancements are available in this update:
 
-No new features and enhancements are available in this update.
+### Integration with AMPLIFY Unified Catalog
+
+This is a Proof of concept (POC), and it is the first reference client for the AMPLIFY Unified Catalog. It establishes connectivity between API Portal and the Unified Catalog using a `DOSA` service account from AMPLIFY.
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20200530_apip_relnotes.md:25]: @jowens-axway  "it is the first reference client" --> what does this mean? Doesn't seem relevant...

Comment (andreamussap) [inline content/en/docs/apim_relnotes/20200530_apip_relnotes.md:25]: @jowens-axway "using a DOSA service account" --> what's **DOSA**? I googled and didn't find anything that I could relate here...

**Comment by andreamussap** [comment]
@jowens-axway I've made some minor rewording to improve readability. Please have a look and let me know if you have any question.

https://deploy-preview-785--axway-open-docs.netlify.app/docs/apim_relnotes/20200530_apip_relnotes/#new-features-and-enhancements

**Note**: I've left two comments (above), please reply to them as well. Thanks.

File: content/en/docs/apim_relnotes/20200530_apip_relnotes.md (hunk 1 above)
Comment (jowens-axway) [reply content/en/docs/apim_relnotes/20200530_apip_relnotes.md:25]: hey andrea, it's actually very relevant as its the first time anyone has ever built anything as a client of the Unified catalog in AMPLIFY.....its a FIRST! defo leave it in here please.

Comment (jowens-axway) [reply content/en/docs/apim_relnotes/20200530_apip_relnotes.md:25]: DOSA's are DevOps Service Accounts used with AMPLIFY to allow machine to machine communication without the need for a standard user account. Further reading on DOSAs are here: https://docs.axway.com/bundle/axway-open-docs/page/docs/central/cli_getstarted/index.html

**Comment by jowens-axway** [comment]
@andreamussap the rest of your updates look great, thanks Andrea!

**Pull request merged**